js/everything.js

```diff
@@ -337,26 +317,22 @@ var ZenPlayer = {
             toggleElement(event, ".plyr__video-wrapper", "Player");
         });
     },
-    setupAutoplayToggle: function()
-    {
+    setupAutoplayToggle: function() {
         // toggle auto next song playing
-        $("#toggleAutoplay").click(function(event)
-        {
+        $("#toggleAutoplay").click(function(event) {
             var toggleTextElement = $("#" + event.currentTarget.id);
-            if (autoplayState === null)
-            {
-                autoplayState = "true";
+            if (!autoplayState) {
+                autoplayState = true;
                 toggleTextElement.text("Stop autoplay");
             }
             else {
-                if (autoplayState === "true")
-                {
+                if (autoplayState === true) {
```

Comment:
don't need to compare a truthy value to `true`, it can just be `if (autoplayState) {`